### ISSUE TITLE
Error in log: ERROR: Root path has no media folders: /media

### ISSUE THREAD

**Issue by michaelsmolka** [advplyr/audiobookshelf#3132]
### What happened?

Hi, although everything seems up and running fine so far (All my audiobooks are there, no unexpected behaviour, etc.) I find the following error in the logs:
```shell
[2024-07-07 20:28:33.259] ERROR: Root path has no media folders: /media (LibraryScanner.js:290)                                                                   
``` 
It appears with every new library scan.

### What did you expect to happen?

Not sure what to type in here, just don't expecting this error. ;-)

### Steps to reproduce the issue

As it appears regularily with every scan, it is not the result of any specific user action.

### Audiobookshelf version

v2.10.1

### How are you running audiobookshelf?

Docker

### What OS is your Audiobookshelf server hosted from?

Linux

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

None

### Logs

```shell
...
[2024-07-07 20:28:32.125] INFO: [LibraryScanner] Starting library scan b7f93c21-8ddf-4399-ae21-93588d333607 for Audiobooks                                        
[2024-07-07 20:28:33.259] ERROR: Root path has no media folders: /media (LibraryScanner.js:290)                                                                   
[2024-07-07 20:28:33.330] INFO: [LibraryScanner] Library scan b7f93c21-8ddf-4399-ae21-93588d333607 completed in 0:01.0 | 0 Added | 0 Updated | 0 Missing          
[2024-07-07 20:28:33.364] INFO: [LibraryController] Scan complete
```

### Additional Notes

I'm running ABS on a Synology DS. The docker container and the media folders are on two separate devices. Media folders are mounted as remote folders via NFS and mapped to the container as following:
```dockerfile
docker run -d \
  -p 13378:80 \
  -v /volume1/docker/audiobookshelf:/config:rw \
  -v /volume1/msNAS/media/audiobooks:/audiobooks:rw \
  -v /volume1/msNAS/media/podcasts:/podcasts:rw \
  -v /volume1/docker/audiobookshelf/metadata:/metadata:rw \
  -e PUID=1026 \
  -e PGID=100 \
  --name Audiobookshelf \
  advplyr/audiobookshelf:latest
``` 
Files are there and even after resetting and restarting the container everything runs as it should (besides this error in the log).
Maybe it's nothing to worry about, but maybe I did something wrong?

**Comment by advplyr** [comment]
Can you console into the docker container and check if the contents of those folders is visible? I don't know about Synology but they may give you an option to open a console for docker containers.

**Comment by michaelsmolka** [comment]
<img width="840" alt="Bildschirmfoto 2024-07-07 um 23 28 41" src="https://github.com/advplyr/audiobookshelf/assets/102899353/99f19a39-67a9-4e08-99c5-1f4029be3baa">
Yes, all the audiobooks (several hundreds) are visible (and playable from the web interface as mobile clients as well).

**Comment by michaelsmolka** [comment]
Sorry, by providing the above details I eventually discovered the reason for the error log: I had during initial setup added an folder "media" to the library but later decided to use the "audiobooks" folder.
The "media" remained unused. But I never looked at the logs so I forgot what I did a few weeks ago. 
Everything is fine now after removing the /media path from the library.
Thanks for your time.
